packages/flutter_driver/test/common.dart

```diff
@@ -25,3 +26,6 @@ void tryToDelete(Directory directory) {
     print('Failed to delete ${directory.path}: $error');
   }
 }
+
+/// Matcher for functions that throw [DriverError].
+final Matcher throwsDriverError = throwsA(isA<DriverError>());
```

Comment:
missing newline at end of file.

Reply:
If I'm not mistaken since #47875 there shouldn't be trailing newline at the end of text files.

Reply:
The file should end with a newline. (That is, the last non-whitespace character should be followed by a single '\n'.) It might be more clear to look at some of the edits in that change. In this file the 'no' symbol that github is displaying means that the single \n is missing.